lib/mobility-core/src/Kernel/Storage/ClickhouseV2/ClickhouseValue.hs

```diff
@@ -35,7 +35,14 @@ class (Show a, Read a) => ClickhouseValue a where
 
 instance ClickhouseValue Double
 
-instance ClickhouseValue Bool
+instance ClickhouseValue Bool where
+  fromClickhouseValue (Value "1") = Just True
+  fromClickhouseValue (Value "0") = Just False
+  fromClickhouseValue (Value "True") = Just True
+  fromClickhouseValue (Value "False") = Just False
+  fromClickhouseValue (Value "true") = Just True
+  fromClickhouseValue (Value "false") = Just False
```

Comment:
`toClickhouseValue` did not changed. It will be used in `where` clauses using `show` function, like:
`SELECT (*) FROM  driver_eda_kafka WHERE  "on_ride"="True"`

Reply:
Ohh yes, ideally we should store as `True` only henceforth, we can discuss on this @hkmangla